k8s-artifacts/K8S-README.md

```diff
@@ -0,0 +1,240 @@
+# WSO2 API Microgateway deployment in Kubernetes
+
+There are two modes when deploying Microgateway in Kubernetes.
+   - Deployment without WSO2 API operator for kubernetes
+   - Deployment with WSO2 API operator for kubernetes
+
+When deploying the Microgateway in kubernetes there are 3 pods for the components as follows.
+   - mg-router
+   - mg-adapter
+   - mg-enforcer
+   
+Router and Adapter services are exposed as NodePort type in kubernetes. Enforcer service is exposed as ClusterIP type.
+
+## Microgateway quick start
+
+*Prerequisites*
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+- [Kubernetes v1.14 or above](https://Kubernetes.io/docs/setup/) <br>
+
+    - Minimum CPU : 6vCPU
+    - Minimum Memory : 6GB
+    
+### Deployment without WSO2 API Operator for kubernetes
+
+Let's host our first API on a Microgateway in kubernetes. We will be exposing the publicly available [petstore services](https://petstore.swagger.io/) via  microgateway in kubernetes
+
+1. First download the CLI tool(APICTL) and the microgateway distribution from the  
+[github release page](https://github.com/wso2/product-microgateway/releases)
+and extract them to a folder of your choice.
+  * [CLI (APICTL)](https://github.com/wso2/product-microgateway/releases)
+  * [Microgateway Distribution](https://github.com/wso2/product-microgateway/releases)
+ 
+  
+CLI tool extracted location will be referred as `CLI_HOME` and Microgateway distribution extracted location would be 
+referred as `MG_HOME`.
+
+2. Using your command line client tool add the 'CLI_HOME' folder to your PATH variable.
+```
+export PATH=$PATH:<CLI_HOME>
+```
+3. Let's deploy the microgateway in kubernetes.
+  - In kubernetes artifacts there are 3 separate folders for each component.
+  - In each folder config maps, deployment and service are included.
+  - You can use the following commands to deploy each microgateway component in kubernetes.
+  ```
+  kubectl apply -f mg-adapter/
+  kubectl apply -f mg-enforcer/
+  kubectl apply -f mg-router/
```

Comment:
Lets merge these 3

Reply:
Fixed in 6b6708a8472501213c00230342810f801d5df4cf